test/e2e_node/gci-init.sh

```diff
@@ -24,25 +24,23 @@ if [ -f "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}" ]; then
   source "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}"
 fi
 
-if [ "${CONTAINERD_CGROUPV2:-"false"}"  == "true" ]; then
-  # check cos image
-  if [ -r /etc/os-release ]; then
-    OS_ID="$(. /etc/os-release && echo "$ID")"
-  fi
-  if [ "${OS_ID}" = "cos" ]; then
-    if ! grep -q 'systemd.unified_cgroup_hierarchy=true' /proc/cmdline && [ "$(stat -fc %T /sys/fs/cgroup/)" != "cgroup2fs" ]; then
-      echo "Setting up cgroupv2"
-
-      mount_path="/tmp/esp"
-      mkdir -p "${mount_path}"
-      esp_partition="/dev/sda12"
-      mount "${esp_partition}" "${mount_path}"
-      sed -i 's/systemd.unified_cgroup_hierarchy=false/systemd.unified_cgroup_hierarchy=true/g' "${mount_path}/efi/boot/grub.cfg"
-      umount "${mount_path}"
-      rmdir "${mount_path}"
-
-      echo "Reconfigured grub; rebooting..."
-      reboot
+if [[ -n "${CONTAINERD_COS_CGROUP_MODE}" ]]; then
+  if uname -a | grep -q cos; then
+    # cgroup_helper was introduced in COS M97, see if it's available first...
+    if which cgroup_helper > /dev/null 2>&1; then
```

Comment:
`which` is a program that itself might not be present, but `command` is a POSIX shell builtin that will be present.  You can use `command -v cgroup_helper` here instead.

Reply:
Thanks, good point, replaced with `command -v` instead